web/source/kmwbase.ts

```diff
@@ -710,7 +710,11 @@ namespace com.keyman {
 
       PKbd = PKbd || this.core.activeKeyboard;
 
-      return com.keyman.osk.VisualKeyboard.buildDocumentationKeyboard(PKbd, argFormFactor, argLayerId, this.osk.computedHeight);
+      // help.keyman.com will set this function in place, so we'll give it priority.
+      let targetHeight = (this['getOskHeight'] as () => number)();
+      targetHeight = targetHeight || this.osk.computedHeight || 200;
+
+      return com.keyman.osk.VisualKeyboard.buildDocumentationKeyboard(PKbd, argFormFactor, argLayerId, targetHeight);
```

Comment:
For comparison:  https://github.com/keymanapp/help.keyman.com/blob/121cab4b14db817a156ce888a8a2cf3546cea452/cdn/dev/js/kbd-docs.js#L137-L150:

```
              // TODO: These hacks are pretty awful
              keyman.getOskWidth = function() {
                return platform == 'desktop' ? 960 :
                        platform == 'phone' ? 520 : 720;
              };


              keyman.getOskHeight = function() {
                return platform == 'desktop' ? 320 :
                        platform == 'phone' ? 240 : 360;
              };
```

```
              var kbd = keyman.BuildVisualKeyboard(keyboardName, 1, platform, name);
```

Since we directly use this on the current site, I gave it top preference.

Reply:
> // TODO: These hacks are pretty awful

🤣 hacks always live the longest!